contracts/Challenge/FutarchyChallenge.sol

```diff
@@ -31,6 +31,7 @@ contract  FutarchyChallenge is ChallengeInterface {
   LMSRMarketMaker public lmsrMarketMaker;                    // MarketMaker for scalar prediction markets
   Registry public registry;                                  // Address of TCR
   uint public winningMarketIndex;                            // Index of scalar prediction market with greatest average price for long token
+  uint public rewardBalance;                                 // Amount of tokens to be rewarded to challenge winner contigent on leftover intial funding liquidity
```

Comment:
spelling: contingent

Reply:
damn I'm horrible with the comment typos!